### PR TITLE
ExaGO v1.4.0, v1.4.1

### PR DESCRIPTION
**Pull request by ashermancinelli**

### CODE REVIEW

**Comment by ashermancinelli** [comment]
@CameronRutherford Would you mind taking a look at this before I submit to upstream? I'd like to add v1.4.1 as well, but we'll have to wait for that PR to be merged.

**Review by alalazo — CHANGES_REQUESTED** [review]
Style tests are failing.

**Comment by ashermancinelli** [comment]
@alalazo i apologize, this was supposed to be a PR on my fork only. I'll reopen when we've finished internal review. Cheers!

File: var/spack/repos/builtin/packages/exago/package.py [hunk 1]
```diff
@@ -65,37 +68,61 @@ class Exago(CMakePackage, CudaPackage):
...
-        args.append(self.define_from_variant('EXAGO_ENABLE_IPOPT', 'ipopt'))
-        args.append(self.define_from_variant('EXAGO_ENABLE_GPU', 'cuda'))
-        args.append(self.define_from_variant('EXAGO_ENABLE_CUDA', 'cuda'))
-        args.append(self.define_from_variant('EXAGO_ENABLE_PYTHON', 'python'))
-        args.append("-DPETSC_DIR='{0}'".format(spec['petsc'].prefix))
+        # NOTE: If building with spack develop on a cluster, you may want to
+        # change the ctest launch command to use your job scheduler like so:
+        #
+        # args.append(
+        #     self.define('HIOP_CTEST_LAUNCH_COMMAND', 'srun -t 10:00'))
```
Comment (cameronrutherford) [inline var/spack/repos/builtin/packages/exago/package.py:93]: This should probably reference the ExaGO variable instead of the HiOp one.

Comment (cameronrutherford) [reply var/spack/repos/builtin/packages/exago/package.py:93]: Also, do we have any documentation about testing a spack installed ExaGO? I don't think we support `spack test` yet, although you might be able to run ctest from the spack build folder if you build with a cache (?)

**Review by cameronrutherford — COMMENTED** [review]
LGTM apart from the mixed reference in comments.

**Pull request closed without merging**